### PR TITLE
Dca 528 locust support for confluence

### PR DESCRIPTION
**Pull request by SergeyMoroz0703**

### CODE REVIEW

File: app/locustio/confluence/requests_params.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# flake8: noqa
```
Comment (ometelytsia) [inline app/locustio/confluence/requests_params.py:1]: why do you need this here?

File: app/locustio/confluence/requests_params.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    data_sets["pages"] = read_input_file(CONFLUENCE_PAGES)
+    data_sets["blogs"] = read_input_file(CONFLUENCE_BLOGS)
+    data_sets["users"] = read_input_file(CONFLUENCE_USERS)
+    data_sets['static-content'] = read_input_file(CONFLUENCE_STATIC_CONTENT)
+
+    return data_sets
+
+
+class BaseResource:
+    resources_file = 'locustio/jira/resources.json'
```
Comment (ometelytsia) [inline app/locustio/confluence/requests_params.py:18]: 'locustio/jira/resources.json' ?

Comment (SergeyMoroz0703) [reply app/locustio/confluence/requests_params.py:18]: Good catch :) will fix it

File: app/locustio/confluence/requests_params.py (hunk 1 above)
Comment (SergeyMoroz0703) [reply app/locustio/confluence/requests_params.py:1]: There is a lot of `W605 invalid escape sequence` pep warning in this file in regular expressions.

**Pull request merged**